guides/common/modules/con_about-running-jobs-on-hosts.adoc

```diff
@@ -11,6 +11,12 @@ Communication occurs through {SmartProxyServer}, which means that {ProjectServer
 Remote execution uses the SSH service that must be enabled and running on the target host.
 Ensure that the remote execution {SmartProxy} has access to port 22 on the target hosts.
 
+Enable and configure the `pull-mqtt` pull mode on the {SmartProxy}.
+When using `pull-mqtt` for pull mode, the remote client initiates all connections, and there are no outgoing connections from the proxy.
+{Project} delegates pull jobs to the proxy, and the proxy notifies the host of a new job over the Message Queueing Telemetry Transport (MQQT).
+The target host then pulls and runs the job.
```

Comment:
We do not describe the communication (or its pull direction) via MQTT broker. The below statements are implicitly said by "the remote client initiates all connections" so there is question if we should be concise or descriptive here. I leave on your decision if the below makes sense to add here or not:

"The target host subscribes to the broker for jobs notifications. Once it receives a notification, it pulls the job details from the proxy and runs the job."